Makefile

```diff
@@ -52,6 +52,8 @@ apply: ## Deploy the controller into your ~/.kube/config cluster
 		$(HELM_OPTS) \
 		--set controller.image=$(shell $(WITH_GOFLAGS) ko build -B github.com/aws/karpenter/cmd/controller) \
 		--set webhook.image=$(shell $(WITH_GOFLAGS) ko build -B github.com/aws/karpenter/cmd/webhook)
+	@# apply any modified/added CRDs to the server
+	@git status charts/karpenter/crds -s | grep "^ [MA]" | sed 's/^ [MA] //' | xargs -n 1 kubectl apply -f
```

Comment:
Gah -- I hate how helm works w/ CRDs. Can we just do `kubectl apply -f charts/karpenter/crds`? The git stuff will only work if you're the one developing the change.

Reply:
Yeah that works, guess it doesn't hurt to always apply.